include/casbin/model/exprtk_config.h

```diff
@@ -108,9 +172,28 @@ namespace casbin {
                 if (type == ExprtkFunctionType::Gfunction) {
                     std::string idenfier(narg, 'S');
                     return std::make_shared<ExprtkGFunction>(idenfier, rm);
-                } else {
-                    return nullptr;
+                } else if (type != ExprtkFunctionType::Unknown) {
+                    std::string idenfier(narg, 'S');
+                    auto ret = std::make_shared<ExprtkOtherFunction>();
+                    if (type == ExprtkFunctionType::KeyMatch) {
```

Comment:
```suggestion
                    switch (type) {
```

Reply:
I have changed. But this commit makes these conflict, sorry about that.
![image](https://user-images.githubusercontent.com/43725202/158052888-024adea4-d0a8-434e-955e-f3c7fc817be3.png)